txscript/scriptbuilder_test.go

```diff
@@ -157,7 +157,7 @@ func TestScriptBuilderAddData(t *testing.T) {
 		name     string
 		data     []byte
 		expected []byte
-		useFull  bool // use AddFullData instead of AddData.
+		useFull  bool // use AddDataUnchecked instead of AddData.
```

Comment:
It's not a big deal since there is a comment right there, but maybe `useFull` should be renamed to something like `useUnchecked` since the terminology is being updated everywhere else?

Reply:
Agreed.  Updated.